### PR TITLE
Phalcon 4 compatibility

### PR DESCRIPTION
**Pull request by sinbadxiii**
Already required compatibility with version 4.
Made some changes.

@sergeyklay what will you do? Will you make tag v2.x.x or will you create a separate repository?

### CODE REVIEW

File: src/Breadcrumbs.php [hunk 1]
```diff
@@ -225,7 +225,7 @@ public function setImplicitFlush($implicitFlush)
      */
     public function setTemplate($linked, $notLinked, $icon)
     {
-        $eventsManager = $this->getEventsManager();
+        $eventsManager = $this->getDI()->getShared('eventsManager');
```
Comment (sergeyklay) [inline src/Breadcrumbs.php]: Could you please change this a bit:
```php
if ($this->getDI()->has('eventsManager')) {
    $eventsManager = $this->getDI()->getShared('eventsManager');
    // ...
}
```

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by sergeyklay** [comment]
It seems we can have compatible code for both versions

**Comment by sinbadxiii** [comment]
and if so?

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**